doc/demo-guide/index.rst

```diff
@@ -1,7 +1,18 @@
 Demo data and project
 =====================
 
-QWAT can be tested by downloading the QGS project repository `here <https://github.com/qwat/QWAT>`_ and restoring the latest `full data dump <https://github.com/qwat/qwat-data-model/releases>`_ in PostgreSQL (+ extension PostGIS and hstore).
+QWAT can be tested by downloading the QGS project repository `here <https://github.com/qwat/QWAT>`_ and restoring the latest `data_and_structure_sample.backup <https://github.com/qwat/qwat-data-model/releases/latest>`_ in PostgreSQL (+ extension PostGIS and hstore).
+
+On the download page you can find theses backups:
+
+- `qwat_vX.X.X_data_and_structure_sample.backup`: Restore a full demo with dataset sample
+
+- `qwat_vX.X.X_structure_only.backup`:  Restore only the structure of QWAT (you need at least demo or custom value_list)
```

Comment:
Suggestion: "Restore the QWAT schema only"

I find the sentence "you need at least demo or custom value_list" a bit confusing. What is "demo" for example? Instead I think I'd say: "in addition you'll also need to restore the value_list_data_only dump".

Reply:
Right. Done